css/index.css

```diff
@@ -98,3 +98,36 @@ header a:not(:last-child) {
     align-items: center;
 }
 
+.img-section-2 {
+    background-image: url(/Users/maksim/Desktop/project/images/photo%20secondary%20section.png);
+    background-repeat: no-repeat;
+    background-position: right top;
+}
+
+.container-section-2 {

```

Comment:
why we need this if we can use .container? and add div inside section with this class

Reply:
in this class, I also added height. Should I make a separate rule for height?